### PR TITLE
[metal3] Pass machine to utilization card

### PR DESCRIPTION
**Pull request by honza**
Instead of using the API to get it

### CODE REVIEW

File: frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/BareMetalHostDashboard.tsx [hunk 1]
```diff
@@ -14,9 +16,11 @@ const BareMetalHostDashboard: React.FC<BareMetalHostDashboardProps> = ({
   machines,
   nodes,
 }) => {
+  const machine = machines.find((m) => getName(m) === getHostMachineName(obj));
```
Comment (jtomasek) [inline frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/BareMetalHostDashboard.tsx]: there is a selector for this:
```suggestion
  const machine = getHostMachine(obj, machines);
```

Comment (honza) [reply frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/BareMetalHostDashboard.tsx]: Done!

**Pull request merged**